packages/elements/src/tree/elements/tree.ts

```diff
@@ -72,7 +72,7 @@ export class Tree<T extends TreeDataItem = TreeDataItem> extends List<T> {
    * @type {TreeRenderer}
```

Comment:
let's remove this `@type` as it's duplicate with type info from TypeScript itself.

Reply:
```suggestion
```